admin/admin-default-term.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Manages filters and actions related to default terms.
+ *
+ * @since 3.0
+ */
+class PLL_Admin_Default_Term {
+
+	/**
+	 * A reference to the PLL_Model instance.
+	 *
+	 * @since 2.8
+	 *
+	 * @var PLL_Model
+	 */
+	protected $model;
+
+	/**
+	 * Preferred language to assign to new contents.
+	 *
+	 * @var PLL_Language
+	 */
+	protected $pref_lang;
+
+	/**
+	 * Reference to Polylang options array
+	 *
+	 * @var array $options
+	 */
+	protected $options;
+
+	/**
+	 * @param object $polylang
```

Comment:
Please add comments in all your methods (description, param short description, `@since`).